src/plugins/sqlalchemy_create_object_operator.py

```diff
@@ -103,16 +107,27 @@ def execute(self, context: Dict[str, Any]) -> None:
         else:
             self.data_table_name = self.data_table_name
 
-        engine = _get_engine(self.db_conn)
+        # setup the database schema for the database connection
+        if self.pg_schema is not None:
+            engine = _get_engine(
+                self.db_conn,
+                pg_schemas=[
+                    self.pg_schema,
+                ],
+            )
+        else:
+            engine = _get_engine(self.db_conn)
```

Comment:
Dit zou nog wat korter kunnen met:

    kwargs = {"pg_schemas": [self.pg_schema]} if self.pg_schema is not None else {}
    engine = _get_engine(self.db_conn, **kwargs)